index.bs

```diff
@@ -2980,9 +2975,11 @@ Note: The {{CollectedClientData}} may be extended in the future. Therefore it's
     ::  This member contains the inverse of the `sameOriginWithAncestors` argument value
         that was passed into the [=internal method=].
 
-    :   <dfn>tokenBinding</dfn>
+    :   \[RESERVED] <dfn>tokenBinding</dfn>        
     ::  This OPTIONAL member contains information about the state of the [=Token Binding=] protocol [[!TokenBinding]] used when communicating
-        with the [=[RP]=]. Its absence indicates that the client doesn't support token binding.
+        with the [=[RP]=]. Its absence indicates that the client doesn't support token binding
+        
+        Note: While [=Token Binding=] was present in Level 1 and Level 2 of WebAuthn, it should not be expected to be present or supported in future versions of the specification.
```

Comment:
Change "it should not be expected to be present or supported in future versions of the specification" to "its use is not expected in Level 3.  The {{CollectedClientData/tokenBinding}} field is reserved so that it will not be reused for a different purpose."